charts/airbyte-webapp/templates/ingress.yaml

```diff
@@ -58,9 +58,9 @@ spec:
                 port:
                   number: {{ $svcPort }}
               {{- else }}
-              serviceName: {{ $fullName }}
+              serviceName: {{ $fullName }}-svc
```

Comment:
Also add this change on Line 57 in the same file.

Reply:
Done, thanks! :)